hokusai/development.yml

```diff
@@ -2,16 +2,18 @@ version: "2"
 services:
   metaphysics:
 {% include 'templates/docker-compose-service.yml.j2' %}
-    environment:
-      - MEMCACHED_URL=metaphysics-memcached:11211
+    command: ["yarn", "start"]
     env_file:
       - ../.env.shared
+      - ../.env.development
       - ../.env
-    ports:
-      - 3000:3000
     depends_on:
       - metaphysics-memcached
+    ports:
+      - 5001:5001
+    volumes:
+      - ../:/app
   metaphysics-memcached:
-    image: memcached
+    image: memcached:1.4.34-alpine
```

Comment:
I am not sure why but the `latest` Memcached would not start. The container would just hang when starting up. Pinning the version we use in prod solved this issue.

Reply:
Good call. I think we always want to try and keep this pinned to production to avoid any unnecessary incompatibility issues when testing locally.